Radiant listing PR

Ektisad25 opened the pull request.

cipig commented:
Hi, thanks for the PR.
Some things though:
You don't have to update the json files in `utils` directory, those are generated later by a script.
Same applies to the icon in `icons`, those are generated by the same script from the icon placed in `icons_original`, so please delete the file in icons and just leave one in `icons_original`.
We recently merged some changes to master branch. Can you please merge latest master into your own repo, so those changes are not reverted in your PR?

Ektisad25 commented:
> Hi, thanks for the PR. Some things though: You don't have to update the json files in `utils` directory, those are generated later by a script. Same applies to the icon in `icons`, those are generated by the same script from the icon placed in `icons_original`, so please delete the file in icons and just leave one in `icons_original`. We recently merged some changes to master branch. Can you please merge latest master into your own repo, so those changes are not reverted in your PR?

alright sir i have removed RXD from Utils files also i removed rxd.png from icons directory

cipig commented:
you could have just remove those files from PR
but the changes to coins file regarding unrelated coins are still there, see https://github.com/KomodoPlatform/coins/pull/827/files
if we would merge the PR as is, it would do those changes
that is because there were new commits to master branch after you cloned it and sent the PR
to get rid of them, you need to merge master branch of this repo into yours

Ektisad25 commented:
i removed these files if not working let me know? PLEASE

cipig commented on coins:
```diff
@@ -5202,8 +5202,7 @@
     "pubtype": 36,
     "p2shtype": 5,
     "wiftype": 128,
-    "txfee": 10000,
```
please don't change unrelated things

Ektisad25 replied:
ok

cipig commented:
> i removed these files if not working let me know? PLEASE

but the unrelated changes in coins file are still there... i left a comment on one place
as i said, it's because of changes to our repo that were done after you cloned it
to get rid of those changes, merge our master branch into yours
eg `git merge upstream/master` on CLI

Ektisad25 commented:
can i create new PR  so will make it correct

cipig commented:
> can i create new PR so will make it correct

sure, that is another option

Ektisad25 commented:
how to get send you my pull request url because this one will be deleted so i can make new

cipig commented:
i will be notified of all new PRs anyway though mail
i am on discord if you want to talk, here is an invite https://discord.gg/YUdC8DeH8D
